arkane/statmech.py

```diff
@@ -226,6 +226,21 @@ def load(self, pdep=False, plot=False):
         species object.
         """
         path = self.path
+        directory = os.path.abspath(os.path.dirname(path))
+        if not os.path.isfile(path):
+            path = os.path.join(directory, path)
+
+        def create_log(log_path):
+            if not os.path.isfile(log_path):
+                modified_log_path = os.path.join(directory, log_path)
+                if not os.path.isfile(modified_log_path):
+                    raise InputError('Could not find single point energy file for species {0} '
+                                     'in the specified path {1}'.format(self.species.label, log_path))
+                else:
+                    log_path = modified_log_path
+
+            return ess_factory(log_path)
+
         is_ts = isinstance(self.species, TransitionState)
         file_extension = os.path.splitext(path)[1]
```

Comment:
Should this technically be `file_extension = os.path.splitext(path)[-1]` for the same reasoning you had about splitting paths in `factory.py`?

Reply:
Yes, I think so. Good catch!